share/functions/__fish_svn_prompt.fish

```diff
@@ -0,0 +1,169 @@
+# colour of the revision number to display in the prompt
+set -g __fish_svn_prompt_color_revision yellow
+
+# setting the prompt status separator character
+set -g __fish_svn_prompt_char_separator "|"
+
+# ==============================
+
+# SVN status symbols
+# Do not change these! These are the expected characters that are output from `svn status`, they are taken from `svn help status`
```

Comment:
It seemed like this was the most efficient way to access them for parsing, I don't like they are global defines but there didn't seem to be a more efficient way of doing this when the helper function they get used in will be called many times.